docs/app/components/components.component.ts

```diff
@@ -1,15 +1,85 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
+import { Router, NavigationEnd } from '@angular/router';
 
 @Component({
     selector: 'hc-demo-components',
     templateUrl: './components.component.html'
 })
-export class ComponentsComponent implements OnInit {
+export class ComponentsComponent {
 
-    constructor() {
-    }
+    thisPage = 'Accordion';
 
-    ngOnInit() {
+    constructor( private router: Router ) {
+        // Listen for vertical tab bar navigation and update the select component
+        router.events.subscribe((event) => {
+            if (event instanceof NavigationEnd) {
+                if ( event.url === '/components/accordion') {
+                    this.thisPage = 'Accordion';
+                } else if ( event.url === '/components/breadcrumbs') {
+                    this.thisPage = 'Breadcrumbs';
+                } else if ( event.url === '/components/button') {
+                    this.thisPage = 'Button';
+                } else if ( event.url === '/components/checkbox') {
+                    this.thisPage = 'Checkbox';
+                } else if ( event.url === '/components/chip') {
+                    this.thisPage = 'Chips';
+                } else if ( event.url === '/components/drawer') {
+                    this.thisPage = 'Drawer';
+                } else if ( event.url === '/components/icon') {
+                    this.thisPage = 'Icon';
+                } else if ( event.url === '/components/list') {
+                    this.thisPage = 'List';
+                } else if ( event.url === '/components/navbar') {
+                    this.thisPage = 'Navbar';
+                } else if ( event.url === '/components/popover') {
+                    this.thisPage = 'Popover';
+                } else if ( event.url === '/components/radio-button') {
+                    this.thisPage = 'Radio Button';
+                } else if ( event.url === '/components/select') {
+                    this.thisPage = 'Select';
+                } else if ( event.url === '/components/subnav') {
+                    this.thisPage = 'Subnavbar';
+                } else if ( event.url === '/components/tabs') {
+                    this.thisPage = 'Tabs';
+                } else if ( event.url === '/components/tile') {
+                    this.thisPage = 'Tile';
+                }
+            }
+        });
     }
 
+    // Handle changes to the select component and navigate
+    selectUpdate ( event: any ) {
```

Comment:
For consideration: 

You could turn this piece into a function like so:
```
    navigateToRoute(path: string) {
        this.router.navigate([path]);
    }
```
Then in the template you can call that function with the appropriate path?